### ISSUE TITLE
NHentai Cloudflare Check Failure

### ISSUE THREAD

**Issue by kschou95** [tachiyomiorg/tachiyomi-extensions#17273]
### Source information

NHentai 1.4.39

### Source language

english, chinese, japanese, multi

### Steps to reproduce

On default user agent string, can't pass cloudflare.
open on webview shows "failure" after ticking the box.

### Expected behavior

shows manga list

### Actual behavior

manga list doesn't show

### Tachiyomi version

0.14.6

### Android version

Android 13

### Other details

same as issue #17224, but here's more checks and fixes that I had tried by following the troubleshooting guide.

1. Loading website in webview. after ticking the box to verify as human, it shows "failure" briefly and goes on a loop.
2. Clearing cookies and webview data. tried, issue doesn't resolve.
3. Changing user agent. Changed to "Mozilla/5.0 (Linux; Android 13) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/115.0.5790.136 Mobile Safari/537.36". Issue for NHentai resolved, but will affect other sources (wnacg/htmanga) as happened in issue #17266.
4. Webview is on version 116.0.5845.51
5. Using on another network with different ISP doesn't resolve. Connecting to Cloudflare Warp+ VPN doesn't resolve.

tldr, if i wanted to use nhentai by changing the user agent, i lose wnacg/htmanga usage, or vice versa.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.14.6](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.
